aod/rotki_assets.ipynb

```diff
@@ -0,0 +1,930 @@
+       "│ eip155:1/e ┆ 0xB6eD7644 ┆ 1     ┆ 0xBitcoin  ┆ … ┆ oxbitcoin  ┆ 0XBTC     ┆ 2018-02-0 ┆ null     │\n",
+       "│ rc20:0xB6e ┆ C69416d67B ┆       ┆            ┆   ┆            ┆           ┆ 6         ┆          │\n",
+       "│ D7644C6941 ┆ 522e20bC29 ┆       ┆            ┆   ┆            ┆           ┆ 00:00:00  ┆          │\n",
+       "│ 6d…        ┆ 4A…        ┆       ┆            ┆   ┆            ┆           ┆           ┆          │\n",
+       "│ eip155:1/e ┆ 0x11111111 ┆ 1     ┆ 1INCH      ┆ … ┆ 1inch      ┆ null      ┆ 2020-12-2 ┆ null     │\n",
+       "│ rc20:0x111 ┆ 1117dC0aa7 ┆       ┆ Token      ┆   ┆            ┆           ┆ 3         ┆          │\n",
+       "│ 111111117d ┆ 8b770fA6A7 ┆       ┆            ┆   ┆            ┆           ┆ 18:13:31  ┆          │\n",
+       "│ C0…        ┆ 38…        ┆       ┆            ┆   ┆            ┆           ┆           ┆          │\n",
+       "│ eip155:1/e ┆ 0x0F72714B ┆ 1     ┆ 1SG        ┆ … ┆ 1sg        ┆ null      ┆ 2018-12-2 ┆ null     │\n",
+       "│ rc20:0x0F7 ┆ 35a366285D ┆       ┆            ┆   ┆            ┆           ┆ 5         ┆          │\n",
+       "│ 2714B35a36 ┆ f85886A2eE ┆       ┆            ┆   ┆            ┆           ┆ 00:00:00  ┆          │\n",
+       "│ 62…        ┆ 17…        ┆       ┆            ┆   ┆            ┆           ┆           ┆          │\n",
+       "│ eip155:1/e ┆ 0xAf30D2a7 ┆ 1     ┆ FirstBlood ┆ … ┆ first-bloo ┆ null      ┆ 2016-07-0 ┆ null     │\n",
+       "│ rc20:0xAf3 ┆ E90d7DC361 ┆       ┆            ┆   ┆ d          ┆           ┆ 9         ┆          │\n",
+       "│ 0D2a7E90d7 ┆ c8C4585e9B ┆       ┆            ┆   ┆            ┆           ┆ 00:00:00  ┆          │\n",
+       "│ DC…        ┆ B7…        ┆       ┆            ┆   ┆            ┆           ┆           ┆          │\n",
+       "│ eip155:1/e ┆ 0xfDBc1aDc ┆ 1     ┆ 1World     ┆ … ┆ 1world     ┆ null      ┆ 2017-11-3 ┆ null     │\n",
+       "│ rc20:0xfDB ┆ 26F0F8f860 ┆       ┆            ┆   ┆            ┆           ┆ 0         ┆          │\n",
+       "│ c1aDc26F0F ┆ 6a5d63b7D3 ┆       ┆            ┆   ┆            ┆           ┆ 00:00:00  ┆          │\n",
+       "│ 8f…        ┆ a3…        ┆       ┆            ┆   ┆            ┆           ┆           ┆          │\n",
+       "└────────────┴────────────┴───────┴────────────┴───┴────────────┴───────────┴───────────┴──────────┘"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05fd359a-3f3a-4b21-b69a-294458ba827c",
+   "metadata": {},
+   "source": [
+    "this database has a long history and since rotki supported for a long time ethereum only most of the tokens that we have in it are ethereum tokens. There are some evm compatible chains for which we started to add assets recently like Celo (42220) that have only one asset"
```

Comment:
```suggestion
    "This asset database has a long history. Since for a long time, rotki supported only ethereum mainnet, most of the tokens that we have in it are ethereum tokens. There are some evm compatible chains for which we started to add assets recently like Celo (42220) that have only one asset."
```